config/routes.rb

```diff
@@ -8,6 +8,16 @@
      resources :likes, only: [:create]
    end
   end
+
+  namespace :api do
+    resources :users, only: [:index, :show] do
+    resources :posts, only: [:index, :new,:create,:show,:destroy] do
+      resources :comments, only: [:index, :new, :create, :destroy]
+      resources :likes, only: [:create]
+    end
+  end    
+ end
```

Comment:
These endpoints don't create the following APIs as required in requirements: 

1. Has API endpoint that returns all post in JSON format - **GET :** `/api/users/:user_id/posts` or `/api/v1/users/:user_id/posts`
1. Has API endpoint that returns all comments for a post - **GET :** `/users/:user_id/posts/:id/comments`
1. Has API endpoint that adds' a comment to a post - **POST :** `/users/:user_id/posts/:id/comments`

Try to put these api routes inside user

Reply:
@descholar-ceo  Thanks for reviewing our Pr. I fixed it now 😄